### PR TITLE
EDLY-3145 Change values position in site configuration to match the on-boarding document 

### PR DESCRIPTION
**Pull request by iam-mhaseeb**
**Description:** Changed values position in site configuration to match the on-boarding document 
**JIRA:** https://edlyio.atlassian.net/browse/EDLY-3145

### CODE REVIEW

File: ecommerce/extensions/edly_ecommerce_app/helpers.py [hunk 1]
```diff
@@ -119,6 +120,7 @@ def user_is_course_creator(request):
     decoded_cookie_data = decode_edly_user_info_cookie(edly_user_info_cookie)
     return decoded_cookie_data.get('is_course_creator', False)
 
+
```
Comment (asadmanzoor93) [inline ecommerce/extensions/edly_ecommerce_app/helpers.py:123]: remove these empty lines added as change.

**Review by asadmanzoor93 — CHANGES_REQUESTED** [review]

**Review by burhandodhy — APPROVED** [review]

File: ecommerce/extensions/edly_ecommerce_app/helpers.py (hunk 1 above)
Comment (iam-mhaseeb) [reply ecommerce/extensions/edly_ecommerce_app/helpers.py:123]: PEP-8 says 2 blank lines between each function call if it's not part of a class. Added those lines to match pep-8 standards that were ignored previously. 
https://www.python.org/dev/peps/pep-0008/#blank-lines

**Review by asadmanzoor93 — APPROVED** [review]

File: ecommerce/extensions/edly_ecommerce_app/tests/test_helpers.py [hunk 2]
```diff
@@ -193,25 +193,24 @@ def test_get_payments_site_configuration(self):
...
             'COLORS': self.request_data.get('colors'),
             'FONTS': self.request_data.get('fonts'),
             'BRANDING': self.request_data.get('branding'),
-            'SESSION_COOKIE_DOMAIN': self.request_data.get('session_cookie_domain'),
             'DJANGO_SETTINGS_OVERRIDE': {
                 'SESSION_COOKIE_DOMAIN': self.request_data.get('session_cookie_domain'),
-                'GTM_ID': '',
                 'OSCAR_FROM_EMAIL': self.request_data.get('oscar_from_email', ''),
                 'LANGUAGE_CODE': 'en',
+                'PAYMENT_PROCESSOR_CONFIG': {},
```
Comment (taimoor-ahmed-1) [inline ecommerce/extensions/edly_ecommerce_app/tests/test_helpers.py:211]: shouldn't we use data from request here as well?

Comment (iam-mhaseeb) [reply ecommerce/extensions/edly_ecommerce_app/tests/test_helpers.py:211]: There's seperate function to test `PAYMENT_PROCESSOR_CONFIG ` which is `test_get_payment_processors_names`

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Pull request merged**